src/db/collections/outputs/mod.rs

```diff
@@ -85,6 +109,24 @@ impl MongoDb {
             )
             .await?;
 
+        collection
+            .create_index(
+                IndexModel::builder()
+                    .keys(doc! { "address": 1 })
```

Comment:
Should be `details.address` now I believe.

Reply:
of course ... you're right ... how could I miss that :see_no_evil: